build.sbt

```diff
@@ -57,6 +57,7 @@ lazy val core = (project in file("core"))
 
       // Test deps
       "org.scalatest" %% "scalatest" % "3.2.9" % "test",
+      "org.scalatestplus" %% "scalacheck-1-15" % "3.2.9.0" % "test",
```

Comment:
Reason for this new dependency is mentioned [here](https://github.com/delta-io/delta/pull/1137/files#r875166431).